### PR TITLE
Feature/gnav uc  2020

### PR DESCRIPTION
**Pull request by maccesar21**
## Global Navigation

- UC 2020 update for special title implementation

- Styles for mobile, tablet, and desktop

### CODE REVIEW

File: src/header/css/header-brand.pcss [hunk 1]
```diff
@@ -70,12 +70,30 @@
...
 	&.-has-image {
 		margin-inline-start: 12px;
+		padding-inline-start: 10px;
+
+		&:before {
+			box-shadow: inset 1px 0 0 0 var(--mono-88);
+			display: inline-block;
+			content: '';
+			height: 30px;
+			left: 0;
```
Comment (paulcpederson) [inline src/header/css/header-brand.pcss:91]: this will be incorrect in rtl, right?

**Comment by paulcpederson** [comment]
Does this address #274 ?

File: src/header/js/header-menus.js [hunk 2]
```diff
@@ -557,9 +557,13 @@ export default ({variant = 'desktop'}) => {
 			}
 
 			if (entry.href && entry.label) {
+				console.log(entry.href);
+				console.log(window.location.hostname);
```
[the remark this answers is not in the document]
Comment (maccesar21) [reply src/header/js/header-menus.js]: Those should be gone.. @paulcpederson

**Comment by MikeTschudi** [comment]
@maccesar21, are the changes to the esri-header-brand* CSS just for UC or permanent?  They negatively affect the [Solution Deployment App](https://github.com/ArcGIS/solution-deployment-app/issues/349), so I'd like to know if I can simply wait until after UC or if I have to plan to work around the changes. Thanks!

**Pull request merged**